controllers/suite_test.go

```diff
@@ -199,3 +213,49 @@ func convertToStructuredResource(path string, out interface{}) (err error) {
 	}
 	return nil
 }
+
+var letters = []rune("abcdefghijklmnopqrstuvwxyz")
+
+func appendRandomNameTo(prefix string) string {
+	return concatToMax(63, prefix, generateString(16))
+}
+
+// generateString generates random alphabetical name which can be used for example as application or namespace name.
```

Comment:
In some of our Knative tests we use: 
```
export SYSTEM_NAMESPACE="${SYSTEM_NAMESPACE:-$(uuidgen | tr 'A-Z' 'a-z')}"
```
However this is not a script another idea would be to use something like what K8s has (https://pkg.go.dev/k8s.io/apiserver/pkg/storage/names):
```
func (simpleNameGenerator) GenerateName(base string) string {
	if len(base) > MaxGeneratedNameLength {
		base = base[:MaxGeneratedNameLength]
	}
	return fmt.Sprintf("%s%s", base, utilrand.String(randomLength))
}
```
See https://pkg.go.dev/k8s.io/apimachinery/pkg/util/rand of how that is used with some prefix as the base.

Reply:
Thanks @skonto! I shamelessly copied the code in 75c6e00, as pulling in the whole apiserver dependency sounds a bit like overkill.